packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -0,0 +1,125 @@
+import { Aliases, AuthInfo } from '@salesforce/core';
```

Comment:
When we create a new module like this we normally expose the methods using an index.ts file. Look at how we do it for other features like diagnostics, notifications, etc.

Reply:
Ah okay, got it